src/mouse.zig

```diff
@@ -31,6 +31,16 @@ const video = @import("video.zig");
 /// This datatype is available since SDL 3.2.6.
 pub const MotionTransformCallback = *const fn (user_data: ?*anyopaque, timestamp: u64, window: ?*C.SDL_Window, id: C.SDL_MouseID, x: ?*f32, y: ?*f32) callconv(.C) void;
 
+/// Enum identifying mouse buttons
+pub const Button = enum(c_int) {
```

Comment:
Now that I think about it, it probably is worthwhile to use a dedicated button enum for handling the SDL keypress events rather than the current bitflag approach. Rebase off of master, and ensure the key down and key up events use this enum like in your version. Though is there a particular reason why this enum is non-exhaustive? Are there other potential mouse buttons unlisted that can be encountered?